### ISSUE TITLE
Leaves shouldnt have TRANSPARENT occlusion

### ISSUE THREAD

**Issue by MiranCZ** [jmc2obj/j-mc-2-obj#226]
I am not sure, if this is intentional or not, but leaves have occlusion set to `TRANSPARENT`, so when you export a tree, only faces of the outside leaves get exported. Which makes trees look wierd. 
Here is an image of some random trees I imported into blender using the latest release:
![Image1](https://github.com/jmc2obj/j-mc-2-obj/assets/98049269/b742a959-48f5-48ef-96de-9825d5afee4d)
And here is a comparison when I leaves have occlusion of `NONE`
![Image2](https://github.com/jmc2obj/j-mc-2-obj/assets/98049269/252f1eb8-7637-4a78-84dd-81614517afba)

**Comment by mmdanggg2** [comment]
You're right, that is more correct to how they are in game. I would change this but that would generate quite a lot more geometry that I'm not sure is going to be good for most people. You can of course change it for yourself if you need the detail.

**Comment by Aspirata** [comment]
Why is this issue still open? I think it can be fixed with this setting
![image](https://github.com/user-attachments/assets/d49fd755-2bae-4083-816a-ed34e617677d)

**Comment by MiranCZ** [comment]
> Why is this issue still open? I think it can be fixed with this setting ![image](https://private-user-images.githubusercontent.com/102383379/386582726-d49fd755-2bae-4083-816a-ed34e617677d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE3MDQ1ODgsIm5iZiI6MTczMTcwNDI4OCwicGF0aCI6Ii8xMDIzODMzNzkvMzg2NTgyNzI2LWQ0OWZkNzU1LTJiYWUtNDA4My04MTZhLWVkMzRlNjE3Njc3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNVQyMDU4MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZjk2MmVmYzFjZjc3YzM1ZTdlNGFhNjU3YmRjNzhhZTFkMDZjZDFkYjQ0ZGNjNGExYzczODgxNTEzMzk0MTc1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.W5q8AyYD9oPiVkRSEL6FU0DSxdAMd-Gb-EVcjyrjHbw)

I am pretty sure that is a different setting, have you confirmed it resolves the issue?

**Comment by Aspirata** [comment]
Oh, sorry, I double checked and it doesn't fix the issue(
